client-management-service-impl/src/main/java/io/mosip/esignet/services/ClientManagementServiceImpl.java

```
@@ -139,8 +136,11 @@ private ClientDetailResponseV2 getClientDetailResponseV2(ClientDetail clientDeta
     private String getClientNameLanguageMapAsJsonString(Map<String, String> clientNameMap, String clientName) {
         if(clientNameMap == null) clientNameMap = new HashMap<>();
         clientNameMap.put(Constants.NONE_LANG_KEY, clientName);
-        JSONObject clientNameObject = new JSONObject(clientNameMap);
-        return clientNameObject.toString();
+        try {
+            return objectMapper.writeValueAsString(clientNameMap);
+        } catch (JsonProcessingException e) {
+            throw new EsignetException(ErrorConstants.UNKNOWN_ERROR);
```

Insteado unknown error can we throw some other like invalid_
client_name like that

---

@sacrana0 lets at least log the exception.